<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlutoSubjectCreatorWindow</class>
 <widget class="QMainWindow" name="PlutoSubjectCreatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>161</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PLUTO Full Assessment Subject Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>142</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Subject ID: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="textSubjID">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Subject Type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbSubjType">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>HEALTHY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>STROKE</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Dominant Side:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbDomLimb">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>LEFT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RIGHT</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Affected Side:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cbAffLimb">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>LEFT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RIGHT</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbCreate">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCancel">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
